charts/newrelic-infrastructure/values.yaml

```diff
@@ -21,24 +21,24 @@ images:
   # -- Image for the New Relic Infrastructure Agent sidecar.
   # @default -- See `values.yaml`
   forwarder:
-    pullPolicy: IfNotPresent
-    tag: 1.29.0
-    repository: newrelic/k8s-events-forwarder
     registry: ""
+    repository: newrelic/k8s-events-forwarder
+    tag: 1.29.0
+    pullPolicy: IfNotPresent
```

Comment:
This change is to make it more readable and also to have the lines aligned to be "regexed".

Reply:
As discussed, it can be a bit error-prone (we can easily forget the order is important) but it is the best we can do with the tools we are using.